analyzer-core/src/main/java/ffb/analyzer/core/queue/PriorityQueueManager.java

```diff
@@ -0,0 +1,71 @@
+package ffb.analyzer.core.queue;
+
+import java.util.concurrent.PriorityBlockingQueue;
+
+public class PriorityQueueHandler{
```

Comment:
Space between bracket and class definition

Reply:
Should this also be named `PriorityQueueManager`?

Reply:
Let's add an [@Singleton](https://docs.oracle.com/javaee/7/api/javax/inject/Singleton.html here so the servlet context knows it's should be a Singleton

Reply:
Updated.